include/z64audio.h

```diff
@@ -0,0 +1,1208 @@
+#ifndef Z64_AUDIO_H
+#define Z64_AUDIO_H
+
+#define MK_CMD(b0,b1,b2,b3) ((((b0) & 0xFF) << 0x18) | (((b1) & 0xFF) << 0x10) | (((b2) & 0xFF) << 0x8) | (((b3) & 0xFF) << 0))
+
+#define NO_LAYER ((SequenceLayer*)(-1))
+
+#define TATUMS_PER_BEAT 48
+
+#define IS_SEQUENCE_CHANNEL_VALID(ptr) ((u32)(ptr) != (u32)&gAudioContext.sequenceChannelNone)
```

Comment:
If those are pointers I suggest using either `void*` or `uintptr_t` instead of a `u32` cast
```suggestion
#define IS_SEQUENCE_CHANNEL_VALID(ptr) ((void*)(ptr) != (void*)&gAudioContext.sequenceChannelNone)
```